xchange-coinbasepro/src/main/java/org/knowm/xchange/coinbasepro/service/CoinbaseProMarketDataServiceRaw.java

```diff
@@ -93,6 +98,60 @@ public CoinbaseProProductBook getCoinbaseProProductOrderBook(CurrencyPair curren
     }
   }
 
+  static Instant lastCall = null;
```

Comment:
I'm not sure this is a thread safe way of accomplishing this. But maybe we solve this in another Merge Request since this is how we did this on GDAX